config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.json

```diff
@@ -14,13 +14,23 @@
         {
           "s-prefix": "schema:contactPoint",
           "properties": [
+            "rdf:type",
             "foaf:firstName",
             "foaf:familyName",
             "schema:telephone",
             "schema:email",
             "schema:jobTitle"
           ]
         },
+        {
+          "s-prefix": "ext:financingPartner",
+          "properties": [
+            "rdf:type",
+            "ext:partnerTitle",
+            "ext:partnerType",
+            "ext:financingAmount"
```

Comment:
The `shacl:order` needs to be added here as well

Reply:
Dont know what this refers to

Reply:
Yeah probably, I think I might have the same issue in projectonderdelen. Will add the shacl:order to the list for you